### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by frosario**

### CODE REVIEW

File: grits.sh [hunk 1]
```diff
@@ -26,15 +28,15 @@ fi
...
-if [[ ! -f grits.tar.gz && ! -f grits.tar ]]; then
-  aws s3 cp s3://bsve-integration/grits.tar.gz ./grits.tar.gz
-  gzip -d grits.tar.gz
-fi
+aws s3 cp s3://bsve-integration/grits.tar.gz ./grits.tar.gz
+gzip -d grits.tar.gz
 
 #Load the image
 docker load < grits.tar
+rm grits.tar
```
Comment (nathanathan) [inline grits.sh:40]: This should be done before the gzip -d command. If a grits.tar is left over from before the update the script will wait for user input to confirm it should be overwritten.

Comment (frosario) [reply grits.sh:40]: I added a line of code to the preliminary cleanup section:
https://github.com/ecohealthalliance/bsve-integration/commit/54fcb58c4b6d0d99320c88990ba812a9e57cd0f2

Comment (nathanathan) [reply grits.sh:40]: @frosario I see the same issue for all the other apps that call gzip -d

Comment (frosario) [reply grits.sh:40]: Yeah shortly after the previous commit, I went through each of the other scripts and applied the same sort of logic in the preliminary section.

Comment (arnoldbm) [reply grits.sh:40]: Is this resolved then?

Comment (frosario) [reply grits.sh:40]: I think so, unless @nathanathan sees any other issue?

**Pull request merged**